### DISCUSSION TITLE
New features made for Vault?

### DISCUSSION THREAD

**Discussion by Sir-IT-Papi** [dani-garcia/vaultwarden#4729] [General]
I recall reading on Monday a comment made about 2 new features made for vw that bw doesn't have and it was followed with op saying new feature like those won't be made for a while, but I can't recall where I saw this. I think it had to do with auto confirm invited users?? 

Can anyone point me towards it? I thought I read it on the latest change log.

**Comment by Sir-IT-Papi** [comment]
My history shows I saw the release at 5:07 and ended up on #754 as I remember reading about it and clicked on the issue, but now what I recall reading isn't on the changelog

![history](https://github.com/dani-garcia/vaultwarden/assets/167458588/a7c41cb2-0d1e-4bc2-8b2f-c47a69340a33)

**Comment by BlackDex** [comment]
Not sure where you would have read that. I can't remember anything like that.

**Reply by Sir-IT-Papi** [reply]
> Not sure where you would have read that. I can't remember anything like that.

Hey hey,
 
thanks for the reply I was going crazy for a bit, but that day I was running on no sleep and I just found what I read which I feel dumb.

Sorry!


![Comment I read](https://github.com/dani-garcia/vaultwarden/assets/167458588/2bd36a0f-5a06-4245-9869-58074ed298a5)

**Reply by BlackDex** [reply]
Well, glad you found it. But those aren't features added to Vaultwarden at all.

**Reply by Sir-IT-Papi** [reply]
All good! I think I was sleep deprived my brain keeps telling me I did read something different haha
